tests/Tests/700__sandbox/701__perfromance_sandbox.robot

```diff
@@ -0,0 +1,81 @@
+*** Settings ***
+Documentation    Test Suite for Performance Sandbox Test cases
+Library         OperatingSystem
+Library         Collections
+Library         Process
+Library         String
+
+
+*** Variables ***
+${NAMESPACE}     openshift-kube-apiserver
+${LABEL_SELECTOR}     app=openshift-kube-apiserver
+${MEMORY_THRESHOLD}    102400
+${PERF_CODE}    go run setup/main.go --users 2000 --default 2000  --custom 0 --username "user" --workloads redhat-ods-operator:rhods-operator --workloads redhat-ods-applications:rhods-dashboard --workloads redhat-ods-operator:cloud-resource-operator --workloads redhat-ods-monitoring:blackbox-exporter --workloads redhat-ods-monitoring:grafana --workloads redhat-ods-monitoring:prometheus <<< y   #robocop:disable
+
+
+*** Test Cases ***
+Verify RHODS Performance For Sandbox Onboarding Process
+    [Documentation]    Performance test for RHODS operator onboard
+    ...    to Sandbox environment
+    [Tags]     ODS-1404
+    ...        Sandbox
+    ...        Performance-Test
```

Comment:
maybe the tag can be named just "Performance"

Reply:
Performance would be bit common that why..i it is okay for me to have test included in it.